.github/workflows/build.yml

```diff
@@ -29,7 +29,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        scala: [2.11.12, 2.12.12]
+        include:
+          - scala: 2.11.12
+            scalaShort: "2.11"
+            overall: 0.0
+            changed: 80.0
+          - scala: 2.12.12
```

Comment:
Let's move to 2.12.17

Reply:
Moved in commit 36655182cec317659ebc332364a63190a6569368.